Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:14.04
```

Comment:
Jeg kan ikke få det til at virke. :-(

➜  html2pdf.it git:(dockerize) ✗ docker build -t html2pdf .          
Sending build context to Docker daemon 70.19 MB
Sending build context to Docker daemon 
Step 0 : FROM dockerfile/nodejs:latest
Pulling repository dockerfile/nodejs
INFO[0005] Error: image dockerfile/nodejs:latest not found